pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high-2020.md.erb

```diff
@@ -19,14 +32,14 @@ theme: responsive
 
 <div style="clear: both;">
 
-<div class="col-50" style="padding-right: 30px; padding-top: 20px; padding-bottom: 50px;">
+<div class="col-50" style="padding-right: 30px; padding-top: 20px; padding-bottom: 25px;">
```

Comment:
Should this file just be HAML at some point? :)

Reply:
Good point.  Content editors could even still write markdown within the HAML file with [the `:markdown` filter](http://haml.info/docs/yardoc/file.REFERENCE.html#markdown-filter), although we might not love that it's a different flavor than the rest of our markdown parsers. (cc @Hamms in case you'd like to loudly object to this!)

Reply:
I don't loudly object to that, but I would strongly recommend instead using partials